docs/src/main/sphinx/installation/containers.rst

```diff
@@ -36,6 +36,15 @@ Run ``docker ps`` to see all the containers running in the background.
 When Trino is still starting, it shows ``(health: starting)``,
 and ``(healthy)`` when it's ready.
 
+.. warning::
+
+   While running the Docker image on Docker can be useful for running Trino
+   locally, it is not a viable solutions for a full-scale production deployment.
```

Comment:
```suggestion
   locally, it is not a viable solution for a full-scale production deployment.
```

Reply:
BTW this leaves room for misinterpretation. Docker images themselves are fine and you can use any non-k8s orchestration as well (or just plain Docker on EC2 as well). Not sure how to clarify "Docker on single machine" vs "Docker container orchestration tools".

Also I'd prefer if this was worded less strictly - after all it's just our recommendation - people are free to run however they want.

Reply:
Docker compose is not single machine